studio/components/interfaces/Auth/SiteUrl/SiteUrl.tsx

```diff
@@ -48,27 +53,32 @@ const SiteUrl = observer(() => {
     }),
   })
 
-  const onSubmit = async (values: any, { setSubmitting, resetForm }: any) => {
+  const onSubmit = (values: any, { setSubmitting, resetForm }: any) => {
     const payload = { ...values }
     payload.DISABLE_SIGNUP = !values.DISABLE_SIGNUP
 
     setSubmitting(true)
-    const { error } = await authConfig.update(payload)
-
-    if (!error) {
-      ui.setNotification({
-        category: 'success',
-        message: `Successfully updated settings`,
-      })
-      resetForm({ values: values, initialValues: values })
-    } else {
-      ui.setNotification({
-        category: 'error',
-        message: `Failed to update settings`,
-      })
-    }
-
-    setSubmitting(false)
+    updateAuthConfig(
+      { projectRef: projectRef!, config: payload },
+      {
+        onError: () => {
```

Comment:
likewise as per https://github.com/supabase/supabase/pull/16227/files#r1289813556
shift all these to the mutation declaration

use isLoading from the mutation rather than isSubmitting from the Form

Reply:
Can't, due to `resetForm`.  I've removed `isSubmitting`.